pkg/api/core/v1/client/apps.go

```diff
@@ -192,8 +193,21 @@ func (c *Client) AppGetPart(namespace, appName, part, destinationPath string) er
 	}
 	defer out.Close()
 
+	contentLenght := response.ContentLength
```

Comment:
```suggestion
	contentLength := response.ContentLength
```

Typo, letter-swap `ht`/`th`

Reply:
Everytime. Thanks!